public/common/js/2fa.js

```diff
@@ -105,7 +105,7 @@
                                                     $("#pass_btn").html("<i class='fa fa-circle-o-notch fa-spin fa-1x fa-fw'></i>Verifying...");
                                                     $.ajax({
                                                         url : "2fa/setupValidate",
-                                                        method : 'POST',
+                                                        method : 'GET',
```

Comment:
why is it changed to get?

Reply:
By mistake. was checking something. Fixed